test/Compiler-caching.test.js

```diff
@@ -72,27 +72,11 @@ describe("Compiler (caching)", function() {
 			});
 		}
 
-		const postCompileCallbackStack = [];
-
-		function addAfterCompileCallback(callback) {
-			postCompileCallbackStack.push(callback);
-		}
-
-		c.plugin("after-compile", (stats, callback) => {
-
-			if(postCompileCallbackStack.length > 0) {
-				postCompileCallbackStack.shift(arguments);
```

Comment:
This handler and `addAfterCompileCallback` are not used. Moreover, the ES6 conversion introduced a bug (`arguments` is wrongly used).